flink-doris-connector/src/main/java/org/apache/doris/flink/catalog/doris/DorisSchemaFactory.java

```diff
@@ -55,14 +54,10 @@ public static TableSchema createTableSchema(
         tableSchema.setKeys(buildKeys(pkKeys, columnFields));
         tableSchema.setTableComment(tableComment);
         tableSchema.setDistributeKeys(buildDistributeKeys(pkKeys, columnFields));
-        tableSchema.setProperties(tableProperties);
-        if (tableProperties.containsKey("table-buckets")) {
-            if (MapUtils.isEmpty(tableBucketMap)) {
-                String tableBucketsConfig = tableProperties.get("table-buckets");
-                tableBucketMap = buildTableBucketMap(tableBucketsConfig);
-            }
-            Integer buckets = parseTableSchemaBuckets(tableBucketMap, table);
-            tableSchema.setTableBuckets(buckets);
+        if (Objects.nonNull(dorisTableConfig)) {
```

Comment:
The logic here seems different? If dorisTableConfig is not empty, it does not necessarily mean that tableBuckets is not empty?

Reply:
Of course, in `DorisSchemaFactory.parseTableSchemaBuckets(
            Map<String, Integer> tableBucketsMap, String tableName)`, it will first be judged whether the tableBuckets is empty.